code/campaignclassic/src/main/java/com/adobe/marketing/mobile/CampaignPushConstants.java

```diff
@@ -43,15 +47,36 @@ private Tracking() {}
 
     final class DefaultValues {
         static final String LEGACY_PAYLOAD_VERSION_STRING = "0";
-        static final int CAROUSEL_MINIMUM_IMAGE_COUNT = 1;
         static final int CAROUSEL_MAX_BITMAP_WIDTH = 300;
         static final int CAROUSEL_MAX_BITMAP_HEIGHT = 200;
         static final String AUTO_CAROUSEL_MODE = "auto";
         static final String FILMSTRIP_CAROUSEL_MODE = "filmstrip";
+        static final int AUTO_CAROUSEL_MINIMUM_IMAGE_COUNT = 1;
+        static final int CENTER_INDEX = 1;
+        static final int FILMSTRIP_CAROUSEL_MINIMUM_IMAGE_COUNT = 3;
+        static final long PUSH_NOTIFICATION_IMAGE_CACHE_EXPIRY = 259200000; // 3 days
```

Comment:
Please prefix expiry with time units (ms, s, min etc..). Also using `TimeUnit.DAYS.toMillis(3);` is more readable.

Reply:
setting the constant to TimeUnit.DAYS.toMillis(3) gives an error:
Static declarations in inner classes are not supported at language level '8'